pytest-embedded-qemu/pytest_embedded_qemu/qemu.py

```diff
@@ -26,6 +28,8 @@ class Qemu(DuplicateStdoutPopen):
     QEMU_STRAP_MODE_FMT = '-global driver=esp32.gpio,property=strap_mode,value={}'
     QEMU_SERIAL_TCP_FMT = '-serial tcp::{},server,nowait'
 
+    QEMU_DEFAULT_QMP = '-qmp tcp:localhost:4488,server,wait=off'
```

Comment:
Oh, I missed it in the last review.

Here if we have multi-dut qemu instances, they will share the same qmp port...

Better to use similar approach like gdb port, we set a `DEFAULT_QMP_TCP_PORT=4488`. For each instance, we use `DEFAULT_QMP_TCP_PORT + self.dut_index` instead.

Reply:
I default values works as suggested, and the port will increase with the `dut_index`.